phivk/gradu.al
# Update/refactos viz to supabase

Note: This depends on #126 and so will need to rebased if there are any changes to that branch

To test:
- Navigate to the index page
- Confirm that the Graph and List view are both displaying the test data from the Supabase instance.
